test/e2e/simple_test.go

```diff
@@ -475,3 +475,136 @@ func TestPersistentVolumeClaim(t *testing.T) {
 	}
 	logger.Infof("Second build finished successfully")
 }
+
+// TestBuildWithSources tests that a build can have multiple similar sources
+// under different names
+func TestBuildWithSources(t *testing.T) {
+	logger := logging.GetContextLogger("TestBuildWithSources")
+	clients := buildClients(logger)
+
+	// Emit a metric for null-build latency (i.e., time to schedule and execute
+	// and finish watching a build).
+	_, span := trace.StartSpan(context.Background(), "NullBuildLatency")
```

Comment:
Remove this, since we should only report this metric for the simple successful build test.